roles/bahmni-reports/templates/bahmni-reports.properties.j2

```diff
@@ -50,4 +50,5 @@ openerp.password= {{ openerp_db_password }}
 bahmnireports.db.url=jdbc:mysql://{{ groups['bahmni-reports-db'][0] }}:3306/bahmni_reports?allowMultiQueries=true
 {% endif %}
 
-reports.save.directory={{ reports_save_dir }}
\ No newline at end of file
+reports.save.directory={{ reports_save_dir }}
+reports.timezone=Asia/Kolkata
```

Comment:
why are we hard-coding timezone here?

Reply:
This is the default time zone.
It is reports time zone as per bahmni implementation. 
So implementer's can modify this as per their time zone.
This could be modified in file /opt/bahmni-reports/etc/bahmni-reports.properties

Reply:
Yes. If this property is empty then default server timezone will be picked up.

Reply:
If so, then maybe we should commit the above `reports.timezone=` as empty, so it picks default server timezone, because that is what we want most of the time. In our case it will automatically be India timezone.

Reply:
Incorporated the review comments. Please review  @gsluthra   @N0-man